ui/react-ui/src/components/bootstrapTable/BootstrapTable.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Table from 'react-bootstrap/Table';
+import Container from 'react-bootstrap/Container';
+
+var BootstrapTable = (props) => {
+  var {heatherItems, rows} = props;
+
+  return (
+    <div className='table-container'>
```

Comment:
Since we are using imported react-bootstrap <Container>, it is ok to remove className='table-container' unless you want to add some custom styling to this specific div. ;)

Reply:
good catch, will change it.